### PR TITLE
Copy email address to clipboard

### PR DESCRIPTION
**Pull request by FabianDevel**
Depends on #629

### CODE REVIEW

File: app/src/main/java/com/infomaniak/mail/ui/main/thread/DetailedContactBottomSheetDialog.kt [hunk 1]
```diff
@@ -58,14 +64,29 @@ class DetailedContactBottomSheetDialog : BottomSheetDialogFragment() {
...
+    }
+
     private fun observeContacts() {
         mainViewModel.mergedContacts.observeNotNull(viewLifecycleOwner) {
             binding.userAvatar.loadAvatar(navigationArgs.recipient, it)
         }
     }
+
+    companion object {
+        private const val CLIP_DATA_LABEL = "Recipient's email"
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/mail/ui/main/thread/DetailedContactBottomSheetDialog.kt:90]:
```suggestion
    private companion object {
        const val CLIP_DATA_LABEL = "Recipient's email"
```

Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/mail/ui/main/thread/DetailedContactBottomSheetDialog.kt:90]:
```suggestion
    private companion object {
        const val CLIP_DATA_LABEL = "Recipient's email"
```

File: app/src/main/java/com/infomaniak/mail/ui/main/thread/DetailedContactBottomSheetDialog.kt [hunk 2]
```diff
@@ -58,14 +64,29 @@ class DetailedContactBottomSheetDialog : BottomSheetDialogFragment() {
         }
 
         addToContacts.setOnClickListener { notYetImplemented() }
-        copyAddress.setOnClickListener { notYetImplemented() }
+        copyAddress.setOnClickListener { copyToClipBoard() }
 
         observeContacts()
     }
 
+    private fun copyToClipBoard() {
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/mail/ui/main/thread/DetailedContactBottomSheetDialog.kt:72]:
```suggestion
        copyAddress.setOnClickListener { copyToClipboard() }

        observeContacts()
    }

    private fun copyToClipboard() {
```

**Review by KevinBoulongne — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/infomaniak/mail/ui/main/thread/DetailedContactBottomSheetDialog.kt (hunk 1 above)
Comment (JorisBodin) [inline app/src/main/java/com/infomaniak/mail/ui/main/thread/DetailedContactBottomSheetDialog.kt:90]: Must be translated

**Review by JorisBodin — CHANGES_REQUESTED** [review]

**Review by KevinBoulongne — APPROVED** [review]

**Review by JorisBodin — APPROVED** [review]

**Pull request merged**